client/src/ui/oAuth.html

```diff
@@ -13,123 +13,117 @@
 
 <body>
 
-	<!-- Masthead -->
-	<header class="masthead text-white text-center">
-		<div class="overlay"></div>
-		<div class="container">
-			<div class="row">
-				<div class="col-xl-9 mx-auto">
-					<h1 class="mb-5">IAM VS code Extension</h1>
-				</div>
-				<div class="col-md-10 col-lg-8 col-xl-7 mx-auto">
-					<form>
-						<div class="col-md-12">
-							<input id="url" input type="url" pattern="https?://.+" class="form-control form-control-lg"
-								placeholder="Enter your IS url" required>
-						</div><br />
-						<div class="col-md-12">
-							<input id="tenant" input type="text" class="form-control form-control-lg"
-								placeholder="Enter your tenant domain" required>
-						</div><br />
-						<div class="col-md-12">
-							<input id="clientID" type="text" class="form-control form-control-lg"
-								placeholder="Client ID" required>
-						</div><br />
-						<div class="col-md-12">
-							<input id="clientSecret" type="password" class="form-control form-control-lg"
-								placeholder="Client Secret" required>
-						</div><br />
-						<div class="col-12 col-md-3 mx-auto">
-							<button id="login" type="submit" class="btn btn-block btn-lg btn-primary">Sign
-								In!</button>
-						</div>
-					</form>
-				</div>
+<!-- Masthead -->
+<header class="masthead text-white text-center">
+	<div class="overlay"></div>
+	<div class="container">
+		<div class="row">
+			<div class="col-xl-9 mx-auto">
+				<h1 class="mb-5">IAM VS code Extension</h1>
+			</div>
+			<div class="col-md-10 col-lg-8 col-xl-7 mx-auto">
+				<form>
+					<div class="col-md-12">
+						<input id="base-url" input type="url" pattern="https?://.+"
+							   class="form-control form-control-lg"
+							   placeholder="Enter your Base Url" required>
+					</div><br />
+					<div class="col-md-12">
+						<input id="clientID" type="text" class="form-control form-control-lg"
+							   placeholder="Client ID" required>
+					</div><br />
+					<div class="col-md-12">
+						<input id="clientSecret" type="password" class="form-control form-control-lg"
+							   placeholder="Client Secret" required>
+					</div><br />
+					<div class="col-12 col-md-3 mx-auto">
+						<button id="login" type="submit" class="btn btn-block btn-lg btn-primary">Sign
+							In!</button>
```

Comment:
I think it is better to move `Sign In!` to a new line between the tags.

Reply:
https://github.com/wso2-extensions/identity-tools-plugin-vscode/pull/7